src/pages/about.mdx

```diff
@@ -6,6 +6,8 @@ title: 'About Me'
 import Experience from '../components/resume/experience.astro'
 import Time from '../components/resume/time.astro'
 
+<div class="post">
```

Comment:
Scattering these `<div>`s is clumsy though. I wonder if that's the trade-off to accept, or if there's a better way.

Reply:
There is an eject button, to override whenever Markdown encounters standard tags like `p` with a custom component. See [this example](https://lukapeharda.com/article/using-tailwindcss-classes-in-markdown/) with Tailwind classes in Astro.

It'd be a lot at the moment to port the global typography styles to that new way of doing things. So I'll stick with this line's CSS cascade for now.